src/dotnet/Chat.UI.Blazor/Components/ChatView/Items/ChatMessage.razor

```diff
@@ -65,9 +65,9 @@
                             </p>
                         }
                     </div>
-                    @if (m.Attachments.Length > 0) {
+                    @if (entry.HasAttachments) {
```

Comment:
Since we now have Attachments collection on ChatEntry, is it not better to get rid of HasAttachments property?

Reply:
done